microsite/src/pages/community/index.tsx

```diff
@@ -42,8 +42,8 @@ const Community = () => {
     {
       title: 'Community sessions',
       content:
-        'Maintainers and adopters meet monthly to share updates, demos, and ideas. Yep, all sessions are recorded!',
-      link: '/on-demand',
+        'Maintainers and adopters meet monthly to share updates, demos, and ideas. You can find recorded session on our YouTube channel!',
```

Comment:
do we have a link to the channel? I couldn't find it

Reply:
https://www.youtube.com/c/BackstageCommunity

Good point - could be interesting to have a link here.